tests/tests_ssh.yml

```diff
@@ -48,18 +61,19 @@
       delegate_to: "{{ kdump_test_ssh_server_outside }}"
       delegate_facts: true
 
-    - include_role:
-        name: linux-system-roles.kdump
-      vars:
-        kdump_ssh_user: >-
-          {{ hostvars[kdump_test_ssh_server_outside]['ansible_user_id'] }}
-        # This is the outside address. Ansible will connect to it to
-        # copy the ssh key.
-        kdump_ssh_server: "{{ kdump_test_ssh_server_outside }}"
-        kdump_path: /tmp/test
-        kdump_target:
-          type: ssh
-          # This is the ssh dump server address visible from inside
-          # the machine being configured. Dumps are to be copied
-          # there.
-          location: "{{ kdump_ssh_user }}@{{ kdump_test_ssh_server_inside }}"
+    - name: >-
+        Fix /sbin/mkdumprd to point netdev to eth0.
+        netdev is the device that mkdumprd must use to connect to the remote
+        server to store kdump logs.
+        On EL 6, mkdumprd does not find netdev correctly when connecting via
+        ssh to localhost.
+      replace:
+        path: /sbin/mkdumprd
+        regexp: netdev=`\/sbin\/ip route get to \$remoteip 2>&1`
+        replace: netdev=eth0
+      when:
+        - ansible_distribution in ['CentOS','RedHat']
+        - ansible_distribution_major_version == "6"
+
+    - name: Run the kdump role and reboot if necessary
+      include_tasks: tasks/execute_with_reboot.yml
```

Comment:
if you are using a task file unconditionally, it is better to use static import (`import_tasks`), because it is evaluated at parse time, not at runtime.

Reply:
Done